docs/gpu-support.md

```diff
@@ -0,0 +1,73 @@
+# Deploying with Accelerators in the Cloud HPC Toolkit
+
+## Supported modules
+
+* [vm-instance]
+* Slurm on GCP modules where applicable, both version 4 and version 5
+  * `schedmd-slurm-gcp-v5-*`
+  * `SchedMD-slurm-on-gcp-*`
+* PBS Pro modules (`pbspro-*`)
+
+## Accelerator definition automation
+
+The [vm-instance] and schedmd-slurm-gcp-v5 [node-group], [controller] and
+[login] modules support automation for defining the `guest_accelerator` config.
+If the user supplies any value for this setting, the automation will be bypassed.
+
+The automation is handled primary in the `gpu_definition.tf` files in these
+modules. This file assumes the existence of two input variables in the module:
+
+* `guest_accelerator`: A list of terraform objects with the attributes type and
+  count.
+* `machine_type`: Defines the machine type of the VM being created.
+
+`gpu_definition.tf` works by checking the `machine_type` to associate it with
+a GPU type and to extract the expected GPU count. The result will always
+define the maximum number of GPUs for the machine_type. If this is not desired,
+the `guest_accelerator` can be defined manually instead.
+
+This automation also only supports machine type `a2`. Machine type `n1` can also
+have guest accelerators attached, however the type and count
+cannot be determined automatically like with `a2`.
+
+[vm-instance]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/modules/compute/vm-instance
+[node-group]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/compute/schedmd-slurm-gcp-v5-node-group
+[controller]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scheduler/schedmd-slurm-gcp-v5-controller
+[login]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scheduler/schedmd-slurm-gcp-v5-login
+
+## Troubleshooting and tips
+
+* To list accelerator types and availability by region, run
+  `gcloud compute accelerator-types list`. The information is also available in
+  the Google Cloud documentation [here](https://cloud.google.com/compute/docs/gpus/gpu-regions-zones).
+* Deployment time of VMs with guest accelerators can be longer than a simple VM.
```

Comment:
nit: I would rephrase as: " Deployment time of VMs with many guest accelerators can take longer."

We should at least suggest how one will increase the timeout for Slurm V5.

Reply:
Added. With the addition of code blocks, I also converted this away from bullets to text.

Reply:
I like how you rewrote this, but I would at least point to the timeouts section so users who just glance at this will get it. Something like:
"See the [Timeouts when deploying a compute VM](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/20937d0bd5f8eabc892739bb6c2705858131e6bb/docs/gpu-support.md#timeouts-when-deploying-a-compute-vm) section below.

Reply:
Done